composio/client/__init__.py

```diff
@@ -616,6 +618,18 @@ def get(  # type: ignore
         actions = actions or []
         apps = apps or []
         tags = tags or []
+        print("Apps - ", apps)
+        print("Actions - ", actions)
+        # Filter out local apps and actions
+        local_apps = [app for app in apps if app.is_local]
+        local_actions = [action for action in actions if action.is_local]
+        apps = [app for app in apps if not app.is_local]
+        actions = [action for action in actions if not action.is_local]
+        only_local_apps = len(apps) == 0 and len(actions) == 0 and (len(local_apps) > 0 or len(local_actions) > 0)
+        if only_local_apps:
+            local_items = self.local_handler.get_list_of_action_schemas(apps=local_apps, actions=local_actions, tags=tags)
+            return [self.model(**item) for item in local_items]
+
         if len(actions) > 0 and len(apps) > 0:
             raise ComposioClientError(
                 "Error retrieving Actions, Both actions and apps "
```

Comment:
WTF does this error even mean?

Reply:
This is mainly gearer towards @kaavee315 
 CC: @shubhras01

Reply:
Did you miss reading the second line or still think the error is not clear? 
`Error retrieving Actions, Both actions and apps cannot be used as filters at the same time.`